niworkflows/interfaces/cifti.py

```diff
@@ -55,14 +61,15 @@
 
 class GenerateCiftiInputSpec(BaseInterfaceInputSpec):
     bold_file = File(mandatory=True, exists=True, desc="input BOLD file")
-    volume_target = traits.Enum("MNI152NLin2009cAsym", mandatory=True, usedefault=True,
+    volume_target = traits.Enum("MNI152NLin2009cAsym", "MNI152NLin6Asym", mandatory=True,
                                 desc="CIFTI volumetric output space")
-    surface_target = traits.Enum("fsaverage5", "fsaverage6", mandatory=True,
-                                 usedefault=True, desc="CIFTI surface target space")
-    subjects_dir = Directory(mandatory=True, desc="FreeSurfer SUBJECTS_DIR")
-    TR = traits.Float(mandatory=True, desc="repetition time")
-    gifti_files = traits.List(File(exists=True), mandatory=True,
-                              desc="list of surface geometry files (length 2 with order [L,R])")
+    surface_target = traits.Enum("fsaverage5", "fsaverage6", "fsLR", mandatory=True,
+                                 desc="CIFTI surface target space")
+    TR = traits.Float(mandatory=True, desc="Repetition time")
```

Comment:
Please, add inputs for density and resolution. Defaults should be ``fsLR``, ``32k``, and 1.6mm.

Reply:
if we want to be in line with 32k, it looks like the 2mm template is used
https://github.com/Washington-University/HCPpipelines/tree/master/global/templates/91282_Greyordinates

Reply:
Updated my comment